demos/android-supabase-todolist/gradle/libs.versions.toml

```diff
@@ -0,0 +1,42 @@
+[versions]
+agp = "8.5.2"
+coreSplashscreen = "1.0.1"
+kotlin = "1.9.0"
+coreKtx = "1.13.1"
+junit = "4.13.2"
+junitVersion = "1.2.1"
+espressoCore = "3.6.1"
+lifecycleRuntimeKtx = "2.6.1"
+activityCompose = "1.9.2"
+composeBom = "2024.04.01"
+materialIconsExtended = "1.7.2"
+uuid = "0.8.2"
+kermit = "2.0.4"
+powersync-sqlite-core = "0.2.1"
```

Comment:
Is this explicitly required? I would imagine this should be a dependency of our Android SDK?

Reply:
You are right, I have removed it